src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,335 @@
+    private GenericEntry          _bedMaximumAngle;
+    private GenericEntry          _maxFlywheelVelocity;
+    private GenericEntry          _climberMaxHeight;
+    private GenericEntry          _maxFlywheelSpeed;
+    private GenericEntry          _FlywheelIntakeSpeed;
+    private final ShooterBed      _ShooterBed;
+    private final Notepath        _notepath;
+    private final Intake          _Intake;
+    private final ShooterFlywheel _ShooterFlywheel;
+    private final Drive           _drive;
+    private final Climb           _climb;
+
+    // SendableChoosers for Autonomous options
+    private SendableChooser<Integer> _autoDelayChooser;
+
+    public Dashboard(ShooterBed shooterBed, Notepath notepath, ShooterFlywheel ShooterFlywheel, Drive drive, Intake intake, Climb climb)
+    {
+        _drive           = drive;
+        _ShooterBed      = shooterBed;
+        _notepath        = notepath;
+        _ShooterFlywheel = ShooterFlywheel;
+        _Intake          = intake;
+        _climb           = climb;
+
+        var tab         = Shuffleboard.getTab("Dashboard");
+        var SettingsTab = Shuffleboard.getTab("Settings");
+
+        // Camera Stream
+        tab.addCamera("Camera Stream", Constants.Vision.CAMERA_NAME, "mjpg:http://10.5.25.12:1181/?action=stream").withWidget("Camera Stream").withPosition(0, 0).withSize(15, 10)
+                .withProperties(Map.of("Show crosshair", false, "Show controls", false));
+        // Field
+        _field = new Field2d();
+        tab.add("field", _field).withPosition(15, 0).withSize(15, 8);
+
+        // Alliance, Note, and Intake boxes
+        var booleanBoxLayout = tab.getLayout("Color Box", BuiltInLayouts.kGrid).withPosition(15, 12).withSize(5, 3).withProperties(Map.of("Number of columns", 1, "Number of rows", 2, "Label position", "LEFT"));
+        _allianceBox = booleanBoxLayout.add("Alliance", false).withPosition(0, 0).withWidget(BuiltInWidgets.kBooleanBox).withProperties(Map.of("Color when true", "blue", "Color when false", "red")).getEntry();
+        _HasNote     = booleanBoxLayout.add("Has Note", false).withPosition(0, 1).withWidget(BuiltInWidgets.kBooleanBox).withProperties(Map.of("Color when true", "green", "Color when false", "red")).getEntry();
+
+        // Swerve Module Angles
+        var driveSettingsLayout = tab.getLayout("Drive Subsystem", BuiltInLayouts.kGrid).withPosition(15, 8).withSize(5, 4);
+        _flAngle = driveSettingsLayout.add("FL Angle", 0.0).withPosition(0, 0).withSize(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _frAngle = driveSettingsLayout.add("FR Angle", 0.0).withPosition(2, 0).withSize(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _blAngle = driveSettingsLayout.add("BL Angle", 0.0).withPosition(0, 2).withSize(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _brAngle = driveSettingsLayout.add("BR Angle", 0.0).withPosition(2, 2).withSize(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
```

Comment:
A few things:
1. Updated dashboard tab name
2. Grid defaults to 3x3, but we want it to be a 2x2
3. If we change the name of the layout, we can shorten the name of the individual entries
4. `withSize` doesn't need to be added to items in a grid layout

```suggestion
        // Swerve Module Angles
        var driveSettingsLayout = dashboard.getLayout("Swerve Angles", BuiltInLayouts.kGrid).withPosition(15, 8).withSize(5, 4).withProperties(Map.of("Number of columns", 2, "Number of rows", 2));
        _flAngle = driveSettingsLayout.add("FL", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
        _frAngle = driveSettingsLayout.add("FR", 0.0).withPosition(1, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
        _blAngle = driveSettingsLayout.add("BL", 0.0).withPosition(0, 1).withWidget(BuiltInWidgets.kTextView).getEntry();
        _brAngle = driveSettingsLayout.add("BR", 0.0).withPosition(1, 1).withWidget(BuiltInWidgets.kTextView).getEntry();
```